hadoop-hdds/docs/content/design/leader-execution/obs-locking.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Ozone Granular locking for OBS bucket
+summary: Granular locking for OBS bucket
+date: 2025-01-06
+jira: HDDS-11898
+status: draft
+author: Sumit Agrawal 
+---
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# OBS locking
+
+OBS case just involves volume, bucket and key. So this is more simplified in terms of locking.
+
+There will be:
+1. Volume Strip Lock: locking for volume
```

Comment:
Strip on what?  I guess the answer is name?  Why name but not id?

Reply:
I have updated doc